thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -150,13 +150,6 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
             maxDiskRecordSize = diskSize->getFixedSize() - fileposSize;
         reportOverflow = false;
     }
-    virtual void setInputStream(unsigned index, CThorInput &_input, bool consumerOrdered) override
-    {
-        PARENT::setInputStream(index, _input, consumerOrdered);
-        // JCSMORE - not sure why you ever want a look ahead on a sink like this?
-        if (!isFastThrough(input))
-            setLookAhead(0, createRowStreamLookAhead(this, inputStream, queryRowInterfaces(input), INDEXWRITE_SMART_BUFFER_SIZE, true, false, RCUNBOUND, this, &container.queryJob().queryIDiskUsage()));
```

Comment:
You might possibly want one if the cost of creating index nodes is significant - might be worth testing with and without.

Reply:
Did you check the time to build an index with and without this?

Reply:
(it may well be faster)